src/ncp/ncp_base.cpp

```diff
@@ -1416,7 +1436,7 @@ ThreadError NcpBase::SendPropertyUpdate(uint8_t header, uint8_t command, spinel_
 
     va_start(args, pack_format);
     SuccessOrExit(errorCode = OutboundFrameBegin());
-    SuccessOrExit(errorCode = OutboundFrameFeedPacked("Cii", header, command, key));
+    SuccessOrExit(errorCode = OutboundFrameFeedPacked(SPINEL_DATATYPE_COMMAND_PROP_S, header, command, key));
```

Comment:
I think what I would like to see is something like the following;

(SPINEL_DATATYPE_HEADER_COMMAND_PROP_S, header, command, prop);

This to me is more intuitive for someone viewing the code for the first time.